.github/workflows/integration-test.yaml

```diff
@@ -1,12 +1,12 @@
-name: "integration-test"
+name: "Integration Test"
 
 on:
   push:
-    # Dependabot has no access to the secrets that this integration test
-    # requires. So we skip running integration tests in branches that dependabot
-    # presumably created.
+    # Dependabot has no access to the secrets that this workflow requires. So we
+    # skip running the workflow in branches that dependabot presumably created.
     branches-ignore:
       - "dependabot/**"
+  workflow_dispatch:
```

Comment:
This could be useful to manually trigger the integration test where we have debug logging enabled.